.github/workflows/ci.yaml

```diff
@@ -389,7 +391,7 @@ jobs:
 
   build-install:
     name: Build current
-    runs-on: ubuntu-latest
+    runs-on: embedded-cluster-2
```

Comment:
is this the large runner or the arm runner? if it's the large runner, it's expensive. any reason we need it?